### DISCUSSION TITLE
CommandInteraction.options excluded in Discordv14

### DISCUSSION THREAD

**Discussion by FordFriedel** [discordjs/discord.js#9285] [Q&A] [answered]
Hello, 
I have been updating my my bot from discord v13 to discord v14 and I cannot figure out how to navigate the lack of an options property on the CommandInteraction class. See it missing here: https://discord.js.org/#/docs/discord.js/14.8.0/class/CommandInteraction

Has the CommandInteractionOptionResolver been moved? I cannot find any comments or notes about it.  Any feedback is appreciated.

**Comment by jaw0r3k** [comment, marked as answer]
CommandInteraction is also for context menus
use `ChatInputCommandInteraction` instead

**Reply by FordFriedel** [reply]
Thanks for the timely response!
